### ISSUE TITLE
Overviewer online

### ISSUE THREAD

**Issue by ArcticFox3107** [overviewer/Minecraft-Overviewer#1654]
Is it possible to run Overviewer in a way that I can upload it somewhere and then other people can see it in an iframe or something. The best way would probably be if I could get it to read off of a file repository that I can auto-update from my computer.

**Comment by enaut** [comment]
Of course thats what overviewer is made for... just upload all the files from the outputdirectory to some host capable of html and you can share it the same as any other website... The only problem is that the maps are rather big.

**Comment by Wunkolo** [comment]
A fully rendered map is just a collection of static files that can be served in any web page. For a while I had it hosted on my small $5/mo VPS but soon enough the map started to get _very_ large(Over 16.9 GiB at the moment) and I had to expand the amount of space the server needed.
The workflow I used was using rclone and ssh to update the remote server's copy of the rendered map with my locally rendered copy. That way, rather than reuploading 100'000 image files, when only a few dozen of them are different, it gets both ends to checksum their local files and only actually transfers what is needed.
